scripts/setup_gae.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This file should not be invoked directly, but called from other Python
+scripts. Python execution environment setup for scripts that require GAE.
+"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import os
+import sys
+import tarfile
+import zipfile
+
+import python_utils
+
+from . import common
+
+_PARSER = argparse.ArgumentParser(description="""
+    This file should not be invoked directly, but called from other Python
+    scripts.
```

Comment:
So why don't we make this a module/move it to common? That way it's impossible to invoke directly?

Reply:
I just copied the description of the original file here. Thinking about it now, I don't see what's the harm in calling it directly. It can be called directly just to set up the environment I guess. I will remove this line from the description